UnicornEngine/video/include/unicorn/video/CameraController.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#ifndef UNICORN_VIDEO_CAMERA_CONTROLLER_HPP
+#define UNICORN_VIDEO_CAMERA_CONTROLLER_HPP
+
+#include <unicorn/utility/SharedMacros.hpp>
+#include <unicorn/video/Camera.hpp>
+#include <memory>
+
+namespace unicorn
+{
+namespace video
+{
+/**
+ * @brief Wrapper for camera, handle shared pointer to it
```

Comment:
For easier creating of new camera controllers. You can inherit it and describe, how you want to "talk" with camera.